content/en/contribute/code/releasing/feature_releases.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: "Feature Releases"
 linkTitle: "Feature Releases"
-weight: 2
+weight: 1
 description: >
     Feature Releases for the CHT Core Framework
+aliases:
```

Comment:
What does this do for the output markup? (no objections, I'm just seeking to learn)

Reply:
As we moved the page and its URL changes, [aliases](https://gohugo.io/content-management/urls/#aliases) help redirect the old URL to the new URL in case someone bookmarked the old link, for example.